pkg/controllers/report/aggregate/controller.go

```diff
@@ -92,12 +92,17 @@ func NewController(
 		logger.Error(err, "failed to register event handlers")
 	}
 	enqueueAll := func() {
-		if list, err := polrInformer.Lister().List(labels.Everything()); err == nil {
+		labelSet := labels.Set{
+			"app.kubernetes.io/managed-by": "kyverno",
```

Comment:
Please use the already existing constants for the label and value

Reply:
@shishir-11 can you address the comment above ?

Reply:
I could not find what constants were needed in the file and would appreciate if someone could point me to them. I'm sorry for the late reply , I was busy with my uni exams.

Reply:
@shishir-11 - please check constants in `api/kyverno/constants.go`.